internal/infrastructure/kubernetes/ratelimit/resource.go

```diff
@@ -182,6 +195,18 @@ func expectedRateLimitContainerEnv(rateLimit *egcfgv1a1.RateLimit, rateLimitDepl
 			Name:  UseStatsdEnvVar,
 			Value: "false",
 		},
+		{
+			Name:  ConfigType,
+			Value: "GRPC_XDS_SOTW",
+		},
+		{
+			Name:  ConfigGrpcXdsServerURL,
+			Value: net.JoinHostPort(XdsGrpcSotwConfigServerHost, strconv.Itoa(XdsGrpcSotwConfigServerPort)),
```

Comment:
Do we need to check if `XdsGrpcSotwConfigServerPort` is a valid port ?

Reply:
right now the `DefaultXdsServerPort` does not have a check either.

https://github.com/envoyproxy/gateway/blob/89b8b398b7c2bf2caa8452f4c72a6228feb2db30/internal/xds/bootstrap/bootstrap.go#L29-L30

maybe we can make these two ports configureable, and done as a follow-up?